addons/account/static/src/components/open_move_line_move_widget/open_move_line_move_widget.js

```diff
@@ -0,0 +1,20 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+import {Many2OneField} from "@web/views/fields/many2one/many2one_field";
+
+class OpenMoveLineMoveWidget extends Many2OneField {
+    setup() {
+        super.setup();
+        this.orm = useService("orm");
+        this.action = useService("action");
+    }
```

Comment:
these are already defined in `Many2OneField`
you can remove the entire `setup()`